conductor_api/src/interface.rs

```diff
@@ -77,6 +91,7 @@ macro_rules! conductor_call {
 /// with spawn() to retrieve the IoHandler.
 pub struct ConductorApiBuilder {
     instances: InstanceMap,
+    instances_map: PublicInstanceMap,
```

Comment:
Could you rename this to something else, or rename `instances` above? It's confusing that `instances` has type `InstanceMap`, and then there's something else called `instances_map`. Maybe `instance_id_map`?

Reply:
will do.